Code/Framework/AzQtComponents/AzQtComponents/Buses/DragAndDrop.h

```diff
@@ -43,60 +49,109 @@ namespace AzQtComponents
      *
      * See http://doc.qt.io/qt-5.8/dnd.html for more complete documentation on QT Drag and Drop.
      *
-     * In general, to use this system, listen on the bus at the address of one of the contexts (see files in AzQtComponents/DragAndDrop/ for contexts)
+     * In general, to use this system, listen on the bus at the address of one of the contexts (see who derives from the above
+     * context base class to see the available contexts).
      * Each time you get any of the events, always check if the event has already been consumed by some other system
-     * by examining the event's "isAccepted()" bool property.  If its already accepted by someone, do not do anything yourself.
+     * by examining the event's "isAccepted()" bool property or the accepted bool passed in.
+     * If its already accepted by someone, do not do anything yourself.
      * otherwise you may examine the mimeData attached in event->mimeData() and decide whether its relevant or not to your handler.
      * if it is, accept the event yourself.
      * The context is intentionally not const, allowing higher priority handlers to fill the context so that later ones can use it.
+     *
+     * Note that this API covers both general dragging (over main windows and other windows of the application such as viewports) as
+     * well as list views, tree views, and other item views.
+     * Note that item views are a special case and the control generally eats the Begin/End/etc events itself.
+     * In that case, the CanDrop and DoDrop handlers are exercised instead of the above.
      */
-    class DragAndDropEvents
+
+    class  DragAndDropEvents
         : public AZ::EBusTraits
     {
     public:
         ///////////////////////////////////////////////////////////////////////
         using BusIdType = AZ::u32;
-        static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::MultipleAndOrdered;
-        static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
+        static constexpr AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::MultipleAndOrdered;
+        static constexpr AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
         ///////////////////////////////////////////////////////////////////////
 
-        virtual int GetPriority() const
+        virtual int GetDragAndDropEventsPriority() const
         {
-            // our default handlers will return 0 for this.  You can override default behavior
-            // by making sure your handler gets invoked first (higher priority).
+            // default fall back 'last ditch effort' handlers will return priority 0.
+            // In general, if you provide specific override behavior and want to suppress the default fallback
+            // behavior, connect a listener with priority > 0.
             return 0;
```

Comment:
AZ::AssetTypeInfo::s_DefaultPriority should be used here and other places in this class.